### DISCUSSION TITLE
To use equals sign or not with options?

### DISCUSSION THREAD

**Discussion by matthewadams** [k3s-io/k3s#9864] [Q&A] [answered]
Real quick.  Is specifying `k3s server --disable=servicelb` produce exactly the same behavior as `k3s server --disable servicelb`?  Note that the former uses the `=` separator between the flag & its value, & the latter uses whitespace.

**Comment by brandond** [comment, marked as answer]
They're equivalent, use whichever form you find more clear. We use standard golang CLI libraries; the code for that is not k3s-specific.

**Comment by matthewadams** [comment]
Thanks, somebody in my company now owes me beer.
